adal/src/main/java/com/microsoft/aad/adal/TokenCacheItem.java

```diff
@@ -82,6 +84,9 @@ public TokenCacheItem() {}
         this.mIsMultiResourceRefreshToken = tokenCacheItem.getIsMultiResourceRefreshToken();
         this.mTenantId = tokenCacheItem.getTenantId();
         this.mFamilyClientId = tokenCacheItem.getFamilyClientId();
+        if (tokenCacheItem.isExtendedLifetimeValid()) {
+            this.mExtendedExpiresOn = tokenCacheItem.getExtendedExpiresOn();
+        }
```

Comment:
shouldn't we just do assign? whoever use it will be responsible for checkign if it's valid?